exist-core/src/main/java/org/exist/xmldb/XmldbURI.java

```
@@ -479,7 +414,7 @@ protected void recomputeURI() throws URISyntaxException {
     protected void safeRecomputeURI() {
         try {
             recomputeURI();
-        } catch (final URISyntaxException e) {
+        } catch (final URISyntaxException ignored) {
```

Why `ignored` when the common syntax is `e`?

---

Because the exception is ignored here, I do like to make it explicit to the reader. Also IntelliJ suggested this change.

---

Okay, I have no strong feeling either way.

---

@line-o I would at least log the exception (even in debug)